docs/administrative/change_account_name.md

```diff
@@ -0,0 +1,64 @@
+---
+permalink: change-account-company-name
+audit_date: '2018-03-13'
+title: Change an account's company name
+type: article
+created_date: '2018-03-13'
+created_by: Rackspace Support
+last_modified_date: '2022-07-04'
```

Comment:
Update date to today or tomorrows date. To whenever you think we will have this merged.